ZFSin/zfs/lib/libpthread/pthread.h

```diff
@@ -809,7 +809,7 @@ static int pthread_create(pthread_t *th, pthread_attr_t *attr, void *(* func)(vo
 	if (attr)
 	{
 		tv->p_state = attr->p_state;
```

Comment:
Thanks. Bit of a personal style, even though "unsigned ssize;" is legal, I'm not a fan of it. Should it accidentally in the future be corrected to "unsigned int ssize;" I would not be objecting :)